gulp/sass.js

```diff
@@ -21,15 +20,6 @@ module.exports = (blueprint, gulp, plugins) => {
         srcGlob: (project, excludePartials) => {
             return path.join(project.cwd, "src/**/", `${excludePartials ? "!(_)" : ""}*.scss`);
         },
-
-        // TODO: make this configurable from root
```

Comment:
done! this is now totally configurable, and there will be a `variables-*` subtask for each package that defines a ` variables` block in Gulpfile.js